fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/JobsServlet.java

```diff
@@ -399,7 +412,12 @@ protected void handleResultsResponse(
     ) {
         HttpResponseMaker httpResponseMaker = new HttpResponseMaker(objectMappers, dimensionDictionary);
 
-        preResponseObservable.flatMap(this::handlePreResponseWithError)
+        preResponseObservable
+                .flatMap(preResponse -> handlePreResponseWithError(
+                        preResponse,
+                        apiRequest.getUriInfo(),
+                        apiRequest.getPaginationParameters())
```

Comment:
The last closing paren on this line (for `handlePreResponseWithError`) should be wrapped onto the next line with the closing parent for `flatMap`.